core/src/main/java/org/opensearch/sql/datasource/model/DataSourceType.java

```diff
@@ -8,5 +8,5 @@
 public enum DataSourceType {
   PROMETHEUS,
   OPENSEARCH,
-  FILESYSTEM
+  JDBC
```

Comment:
FILESYSTEM supported has been removed in https://github.com/opensearch-project/sql/pull/1284.